### PR TITLE
Prep 9701 pdf 186879483

### PR DESCRIPTION
**Pull request by vrajmohan**

### CODE REVIEW

File: src/main/java/org/mdbenefits/app/preparers/ApplicantDetailsPreparer.java [hunk 1]
```diff
@@ -23,34 +23,18 @@ public Map<String, SubmissionField> prepareSubmissionFields(Submission submissio
...
-        // If they said they don't want us to collect this info, don't pass it on, even if they had previously set it.
-        // This check helps if they went back and forward in the pages changing values. The data that prepares these fields is
-        // the default OneToManyPreparer in the ffl. This code just overwrites the fields to _unset_ them.
-        if (submission.getInputData().getOrDefault("permissionToAskAboutRace", "false").equals("false")) {
-            results.put("raceSelected", new CheckboxField("raceSelected", List.of(), null));
-            results.put("ethnicitySelected", new SingleField("ethnicitySelected", "", null));
-        }
+        results.put("applicantDOB", new SingleField("applicantDOB", (String) dob, null));
+
+        results.put("applicantSSN", new SingleField("applicantSSN", SubmissionUtilities.formatSSN((String) inputData.get("encryptedSSN")), null));
```
Comment (bseeger) [inline src/main/java/org/mdbenefits/app/preparers/ApplicantDetailsPreparer.java:36]: I think "encryptedSSN" is not the field you want here. I think you want just "SSN" or whatever it's called in it's decrypted form.

File: src/main/java/org/mdbenefits/app/preparers/HouseholdDetailsPreparer.java [hunk 2]
```diff
@@ -19,68 +19,27 @@
...
-                String uuid = (String) householdMember.get("uuid");
-                results.put("householdUSCitizen" + i, new SingleField("householdUSCitizenDerived",
-                        nonCitizens != null && nonCitizens.contains(uuid) ? "No" : "Yes", i + 1));
+                results.put("householdMemberDOB_" + i, new SingleField("householdMemberDOB", (String) dob, i + 1));
 
-                if (submission.getInputData().getOrDefault("permissionToAskAboutRace", "false").equals("true")) {
-                    results.put("householdRaceEthnicityCode" + i,
-                            new SingleField("householdRaceEthnicityCode", getRaceEthnicityCode(submission, uuid), i + 1));
-                }
+                results.put("householdMemberSSN_" + i, new SingleField("householdMemberSSN", SubmissionUtilities.formatSSN((String) householdMember.get("householdMemberEncryptedSSN")), i + 1));
```
Comment (bseeger) [inline src/main/java/org/mdbenefits/app/preparers/HouseholdDetailsPreparer.java:40]: same here for the "householdMemberEncryptedSSN" - you'll want the unencrypted version.  Unless we are working with the encrypted version of the SSN? That doesn't make sense though.

File: src/main/java/org/mdbenefits/app/preparers/ApplicantDetailsPreparer.java (hunk 1 above)
Comment (vrajmohan) [reply src/main/java/org/mdbenefits/app/preparers/ApplicantDetailsPreparer.java:36]: `encryptedSSN` made my test work. Let me see what happens with the UI flow.

Comment (vrajmohan) [reply src/main/java/org/mdbenefits/app/preparers/ApplicantDetailsPreparer.java:36]: Neither `ssn` not `encrypedSSN` work with the UI flow. I'll add a story to revisit this.

**Review by bseeger — APPROVED** [review]
LGTM and worked as expected for the fields currently mapped.

**Pull request merged**